frontend/src/container/Tracim.jsx

```diff
@@ -301,6 +307,30 @@ export class Tracim extends React.Component {
     }
   }
 
+  handleHeadTitleAndFavicon = (prevHeadTitle, prevNotificationNotReadCount) => {
+    const { props } = this
+
+    const hasHeadTitleChanged = prevHeadTitle !== props.system.headTitle
+    const hasNotificationNotReadCountChanged =
+      props.notificationPage.notificationNotReadCount !== prevNotificationNotReadCount
+    const notificationNotReadCount = props.notificationPage.notificationNotReadCount
+
+    if ((hasHeadTitleChanged || hasNotificationNotReadCountChanged) && props.system.headTitle !== '') {
+      let newHeadTitle = props.system.headTitle
+      if (notificationNotReadCount > 0) {
+        newHeadTitle = `(${notificationNotReadCount > 99 ? '99+' : notificationNotReadCount}) ${newHeadTitle}`
+      }
+      document.title = newHeadTitle
+    }
+
+    if (
+      !hasNotificationNotReadCountChanged ||
+      (prevNotificationNotReadCount > 1 && notificationNotReadCount > 1)
```

Comment:
Because for example if the `notificationNotReadCount` is updated to 12 from 13 the the favicon will not change.
The favicon need to be updated only if `notificationNotReadCount === 0 && prevNotificationNotReadCount > 0` or `notificationNotReadCount > 0 && prevNotificationNotReadCount === 0`

Reply:
Ah ok, make sense. Thank you for the answer :smile: